www/src/pages/en/usage/next-auth.md

```diff
@@ -176,7 +176,30 @@ If for example, you'd like to add a `role` to the `User` model, you would need t
 
 ## Usage with Next.js middleware
 
-Usage of NextAuth.js with Next.js middleware [requires the use of the JWT session strategy](https://next-auth.js.org/configuration/nextjs#caveats) for authentication. This is because the middleware is only able to access the session cookie if it is a JWT. By default, Create T3 App is configured to use the **default** database strategy, in combination with Prisma as the database adapter.
+Usage of NextAuth.js with Next.js middleware [requires the use of the JWT session strategy](https://next-auth.js.org/configuration/nextjs#caveats) for authentication. This is because the middleware is only able to access the session cookie if it is a JWT. By default, Create T3 App is configured to use the **default** database strategy, in combination with Prisma as the database adapter.  
+After switching to the JWT session strategy. Make sure to update the `session` callback in your `[...nextauth].ts`.  
```

Comment:
It should not be `src/server/auth.ts` ?
In new version of the create-t3-app, all the auth configuration stuff is outside of the `[...nextauth].ts`

Reply:
Sure! Wow, I started using T3 app just a week ago. 😅 
```suggestion
After switching to the JWT session strategy. Make sure to update the `session` callback in your `src/server/auth.ts`.  
```